core/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@braintrust/core",
-  "version": "0.0.6",
+  "version": "0.0.45",
```

Comment:
Is this intended to be kept in sync with `version.py` now? I think that file is still on version `0.0.44`.

Reply:
Nope. This just allows autoevals to utilize some of the new features in core. But I might be able to revert this too.